share/spice/go_watch_it/go_watch_it.js

```diff
@@ -1,6 +1,132 @@
 (function (env) {
     "use strict";
 
+    // Private functions, variables and helpers
+    var streaming_providers = {
+        "YouTube": 1,
+        "Google Play": 1,
+        "Disney Movies Anywhere": 1,
+        "Target Ticket": 1,
+        "Hulu": 1,
+        "Crackle": 1,
+        "Flixster": 1,
+        "Netflix": 1
+    };
+
+    var purchase_providers = {
+        "Best Buy": 1,
+        "Walmart": 1,
+        "Target": 1,
+        "Fandango": 1,
+        "Amazon": 1,
+        "Redbox": 1
+    };
+
+    // For retina screen return optimized images @2x.png
+    var append = ((DDG.is3x || DDG.is2x) ? "@2x.png" : ".png");
+
+    var path = "assets/";
+
+    var skip_providers = {
+        // Movies on Demand
+        "10": 1,
+        // Filmmaker Direct Streaming
+        "46": 1,
+        // DirecTV
+        "113": 1
+    };
+
+    var provider_icons = {
+        "1": {
+            dark: DDG.get_asset_path('go_watch_it', path + 'netflix'),
```

Comment:
It doesn't seem like there's any need for us to call `DDG.get_asset_path` in each of these? What about an array, and we call `DDG.get_asset_parth()` below, when the path is actually needed?

Something like...

```js
var providers = {
 1: [ 'netflix', 'netflix'],
 2: [ 'netflix', 'netflix'],
 3: [ 'itunes', 'itunes-alt'],
 ...
};
```